README.md

```diff
@@ -328,6 +329,13 @@ function shouldRenderSuggestions(value) {
 }
 ```
 
+<a name="alwaysRenderSuggestionsProp"></a>
+#### alwaysRenderSuggestions (optional)
+
+By default, suggestions are rendered only if the input field is under focus.
+
+If you'd like to _always_ display suggestions, set `alwaysRenderSuggestions={true}`.
```

Comment:
Shall we also add a note in the `shouldRenderSuggestions` section along the lines of:
> When `shouldRenderSuggestions` returns `true`, suggestions will be rendered only when the input field is focused. If you would like to render suggestions regardless of whether the input field is focused or not, set `alwaysRenderSuggestions={true}` (`shouldRenderSuggestions` is ignored in this case).

?